opsdroid/database/mongo/tests/test_mongo.py

```diff
@@ -35,24 +43,22 @@ async def test_connect(database):
 
 
 @pytest.mark.asyncio
-@pytest.mark.parametrize("config", [{}])
+@pytest.mark.parametrize("config", [{"collection": "test_collection"}])
 async def test_get(database):
-    database.database = {}
-    database.database["test_db"] = DatabaseMongoTest({})
+    database.database = {"test_collection": DatabaseMongoTest({})}
     with pytest.raises(TypeError):
         await database.get("test_db")
 
 
 @pytest.mark.asyncio
-@pytest.mark.parametrize("config", [{}])
+@pytest.mark.parametrize("config", [{"collection": "test_collection"}])
 async def test_put(database):
```

Comment:
To be honest, I'm not sure what I'm doing wrong on this test. Any help would be appreciated.